front-end/src/pages/ProposalEdit/ProposalEdit.js

```diff
@@ -0,0 +1,648 @@
+        },
+        "& .MuiInput-underline:hover:not(.Mui-disabled):before": {
+            borderBottom: "none"
+        },
+        "& .makeStyles-inputLabel-118": {
+            color: "#ffffff",
+            fontSize: "25px"
+        }
+    },
+    inputTitle: {
+        color: "white",
+        width: "200px",
+        fontSize: "32px",
+        disableUnderline: true
+    },
+    inputLabelTitle: {
+        color: "white",
+        fontSize: "25px",
+    },
+    image: {
+        width: 450,
+        height: 294,
+    },
+    img: {
+        margin: 'auto',
+        display: 'block',
+        maxWidth: '100%',
+        maxHeight: '100%',
+    },
+    paper: {
+        padding: theme.spacing(2),
+        margin: 'auto',
+        maxWidth: 450,
+        marginRight: 25
+    },
+    currencyInput: {
+        width: '80%'
+    }
+}));
+
+const HearderCustomizeStar = withStyles({
+    root: {
+        color: '#FFFFFF',
+        width: '28px',
+        height: '28px',
+        marginBottom: '6px'
+    }
+})(Stars)
+
+const TitleCategoryTypography = withStyles({
+    root: {
+        color: '#599C6D',
+        fontWeight: 500,
+        fontSize: '15px',
+    }
+})(Typography);
+
+const CategoryCheckbox = withStyles({
+    root: {
+        color: '#599C6D',
+        fontSize: '12px',
+        '&$checked': {
+            color: '#599C6D',
+        },
+    },
+    checked: {},
+})((props) => <Checkbox color="default" {...props} />);
+
+const MainTitleTyography = withStyles({
+    root: {
+        fontSize: '20px',
+        color: 'rgba(18, 97, 163, 1)',
+        lineHeight: '26.6px',
+        fontWeight: '600'
+    }
+})(Typography);
+
+const GovernmentTitleTyography = withStyles({
+    root: {
+        fontSize: '12px',
+        color: 'rgba(0, 0, 0, 0.5393)',
+        lineHeight: '16x',
+        fontWeight: '400'
+    }
+})(Typography);
+
+const GovernmentContentMiddleTyography = withStyles({
+    root: {
+        fontSize: '14px',
+        color: 'rgba(0, 0, 0, 1)',
+        lineHeight: '16.41px',
+        fontWeight: '400'
+    }
+})(Typography);
+
+const GovernmentContentSmallTyography = withStyles({
+    root: {
+        fontSize: '12px',
+        color: 'rgba(0, 0, 0, 1)',
+        lineHeight: '16.41px',
+        fontWeight: '400'
+    }
+})(Typography);
+
+const CollapseTyography = withStyles({
+    root: {
+        fontSize: '12px',
+        color: 'rgba(18, 97, 163, 1)',
+        lineHeight: '16px',
+        fontWeight: '400'
+    }
+})(Typography);
+
+const CategoryPeopleIcon = withStyles({
+    root: {
+        color: '#000000',
+        fontSize: 15,
+    }
+})(NaturePeople);
+
+const CatergoryItemTyography = withStyles({
+    root: {
+        fontSize: '12px',
+        color: 'rgba(1, 1, 1, 1)',
+        lineHeight: '14.06px',
+        fontWeight: '400'
+    }
+})(Typography);
+
+export default function ProposalDetail() {
+    const classes = useStyles();
+    const [stateCheckBox, setStateCheckBox] = useState({
```

Comment:
This also should be dynamic. Ideally we get this categories from API but for the sake of simplicity, we can hard code them in some helper file and reuse them across all files.

Can we move this hard coded checked values to a helper file and use them across all files ?